TestNodeTick: elapsed = 0, want 1

via https://semaphoreci.com/coreos/etcd/branches/pull-request-7373/builds/3
```
--- FAIL: TestNodeTick (0.03s)
	node_test.go:408: elapsed = 0, want 1
```

--- a/raft/node_test.go
+++ b/raft/node_test.go
@@ -402,7 +402,11 @@ func TestNodeTick(t *testing.T) {
 	go n.run(r)
 	elapsed := r.electionElapsed
 	n.Tick()
-	testutil.WaitSchedule()
+
+	for len(n.tickc) != 0 {
+		time.Sleep(100 * time.Millisecond)
+	}
+
 	n.Stop()
 	if r.electionElapsed != elapsed+1 {
 		t.Errorf("elapsed = %d, want %d", r.electionElapsed, elapsed+1)

